doc/releases/changes.rst

```diff
@@ -3,17 +3,12 @@
 Release Notes
 =============
 
-Version 5.1.0 (unreleased)
+Version 5.0.1 (unreleased)
 --------------------------
 
-Enhancements
-~~~~~~~~~~~~
-
-Bug Fixes
-~~~~~~~~~
-
-Backward-Incompatible Changes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Migrate altair.sphinxext.altairplot module to `sphinxext-altair <https://github.com/altair-viz/sphinxext-altair>`_ extension (#3056).
```

Comment:
```suggestion
```
I actually already added a similar note to the Version 5 changes in this file and the release notes on GitHub.

Reply:
Didn't notice. Even better.